bitcoin/src/address/error.rs

```diff
@@ -6,6 +6,7 @@ use internals::write_err;
 
```

Comment:
In commit message "a segwit key should never be compressed" - never uncompressed, actually.

And FTR, I want to enforce it in the type system, I just didn't manage to make a PR yet.

Reply:
> In commit message "a segwit key should never be compressed" - never uncompressed, actually.

I hope that was a typo and not a brain fail :)

> And FTR, I want to enforce it in the type system, I just didn't manage to make a PR yet.

Me too.